plugins/nodepools.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: nodepools
+spec:
+  version: v0.0.1
+  homepage: https://github.com/grafana/kubectl-nodepools
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/grafana/kubectl-nodepools/releases/download/v0.0.1/kubectl-nodepools_v0.0.1_darwin_amd64.tar.gz
+    sha256: 039d171ac903d909d4c7ec8c55955d64d26ac15dd4955a76e3a8661ff293cc4c
+    bin: kubectl-nodepools
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: arm64
+    uri: https://github.com/grafana/kubectl-nodepools/releases/download/v0.0.1/kubectl-nodepools_v0.0.1_darwin_arm64.tar.gz
+    sha256: c03dfebf358ffbe031595d5ec3d55655f5ef98f906d6a9af57dc95746e772190
+    bin: kubectl-nodepools
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/grafana/kubectl-nodepools/releases/download/v0.0.1/kubectl-nodepools_v0.0.1_linux_amd64.tar.gz
+    sha256: 1c9e710999fcd6f37932c0480f5c0a9870bff968fe5d5d6b2fec932af9141956
+    bin: kubectl-nodepools
+  - selector:
+      matchLabels:
+        os: linux
+        arch: arm64
+    uri: https://github.com/grafana/kubectl-nodepools/releases/download/v0.0.1/kubectl-nodepools_v0.0.1_linux_arm64.tar.gz
+    sha256: 4160b95d11a82ee44956b1eacf65d758b0ef89f08668321746f31e1d66ee901a
+    bin: kubectl-nodepools
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/grafana/kubectl-nodepools/releases/download/v0.0.1/kubectl-nodepools_v0.0.1_windows_amd64.zip
+    sha256: ebacb232dfd97ddb6ad5e4b710b72e3440e918a5ac1b8836341af75b87e62435
+    bin: kubectl-nodepools.exe
+  - selector:
+      matchLabels:
+        os: windows
+        arch: arm64
+    uri: https://github.com/grafana/kubectl-nodepools/releases/download/v0.0.1/kubectl-nodepools_v0.0.1_windows_arm64.zip
+    sha256: eeca0e865046bab307364fb303edc096f1e708a7fd4be63f40f736547dfa5bf7
+    bin: kubectl-nodepools.exe
+  shortDescription: List node pools/groups.
+  description: |
+    List node pools/groups in the current cluster, alongside a count of how
+    many nodes there are in each pool/group and their type.
+
+    You can also list nodes for a given node pool/group by name.
+
+    This plugin supports the standard kubectl flags for environment selection
+    and authentication.
+
+    Example usage:
+      List node pools/groups in cluster:
+        kubectl nodepools
+      List nodes in a node pool/group:
+        kubectl nodepools nodes $nodepool
+
+    More information at https://github.com/grafana/kubectl-nodepools
```

Comment:
no need for example usage. users will run -h/--help
plus we print `kubectl nodepools -h` and the website after installation and the `krew info` command.
typically this info goes out of date, so I recommend omitting.

Reply:
I see, that makes sense. I just borrows inspiration from other plugins (kudo I believe), I'll take it into account for an upcoming update!

Reply:
I can merge as is but I recommend fixing.

Also it'd be cool for maybe the plugin to work without any args and print all nodes grouped by pool? (Sorry I havent used it yet, if that's the case already)

Reply:
I believe you merged it already 🙀

The plugin without any arguments just return a list of node pools/groups. In fact the `list` subcommand is optional. The `nodes` subcommand is an extension that upon a given node pool/group it returns the nodes that are part of it.

Reply:
uh oh lol , I guess /approve is an org membership thing. feel free to send a PR with the fixes.

unrelated: I think people like tree views so having a cluster-wide view/stats of node pools would be extremely useful (maybe also with some stuff like their differing node annotations)

Reply:
> uh oh lol , I guess /approve is an org membership thing. feel free to send a PR with the fixes.

Sure, I'll send another PR later today. Thanks!

> unrelated: I think people like tree views so having a cluster-wide view/stats of node pools would be extremely useful (maybe also with some stuff like their differing node annotations)

In our use case we definitely use the two current views for different purposes, however, that sounds like a nice adding as a flag, something like `kubectl nodepools list -tree` or similar. I'll think about it!